hr_holidays_leave_auto_approve/models/hr_holidays.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 ONESTEiN BV (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class HrHolidays(models.Model):
+    _inherit = "hr.holidays"
+
+    @api.model
+    def create(self, values):
```

Comment:
when redefining  `create()` in the new API you also need to use the  `api.returns` as is done in https://github.com/odoo/odoo/blob/9.0/openerp/models.py#L4118 otherwise you will break other addons extending the same model using the old api.